### PR TITLE
doc(simpleJsp): add Simple JSP Portlet doc with examples.

### PR DESCRIPTION
**Pull request by bjagg**

### CODE REVIEW

File: docs/simple-jsp.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        <name>SimpleJspPortletController.jspName</name>
+        <readOnly>false</readOnly>
+        <value>myJsp</value>
+    </portlet-preference>
+</portlet-definition>
+```
+
+The corresponding example JSP, `overlays/jasig-widget-portlets/src/main/webapp/WEB-INF/jsp/myJsp.jsp`:
+
+```html
```
Comment (ChristianMurphy) [inline docs/simple-jsp.md]:
```suggestion
```jsp
```

File: docs/simple-jsp.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+```$xslt
+overlays/jasig-widget-portlets/src/main/webapp/WEB-INF/jsp/
+```
+
+
+### <a name="examples"></a>Examples
+An example of a portlet definition file, `data/quickstart/portlet-definition/myjsp.portlet-definition.xml`:
+
+```$xslt
```
Comment (ChristianMurphy) [inline docs/simple-jsp.md]:
```suggestion
```xslt
```

or

```suggestion
```xml
```

File: docs/simple-jsp.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    <user-attribute>
+        <description>Username attribute name for uPortal user</description>
+        <name>title</name>
+    </user-attribute>
+```
+
+### <a name="jspdir"></a>JSP Location
+JSPs for the Simple JSP Portlet should be placed in the following location in _uPortal-start_:
+
+```$xslt
```
Comment (ChristianMurphy) [inline docs/simple-jsp.md]:
```suggestion
```xslt
```

or

```suggestion
```xml
```

File: docs/simple-jsp.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+in these JSPs. Often, this file needs to be copied from this project to appropriate location
+in _uPortal-start_'s overlay of `jasig-widget-portlets`:
+
+```
+overlays/jasig-widget-portlets/src/main/webapp/WEB-INF/portlet.xml
+```
+
+Add more user attributes as desired, after the current `user.login.id` is defined:
+
+```$xslt
```
Comment (ChristianMurphy) [inline docs/simple-jsp.md]:
```suggestion
```xslt
```

or

```suggestion
```xml
```

**Comment by jgribonvald** [comment]
Else why not also documenting the jsp Invoker ?

**Comment by bjagg** [comment]
@jgribonvald Good idea. It's something to add to the list of docs to write!

**Comment by bjagg** [comment]
@ChristianMurphy Thanks for the review!

**Pull request merged**